OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
 	Thx - Hardy
 
 -->
-    <string name="permission_is_required">Permission is required to use this option.</string>
-    <string name="logcat_buffer_descr">Check and share detailed logs of the application</string>
+    <string name="permission_is_required">Please grant the required permission to use this option.</string>
+    <string name="logcat_buffer_descr">Check and share detailed logs of the app</string>
     <string name="file_does_not_contain_routing_rules">No routing rules in \'%1$s\'. Please choose another file.</string>
     <string name="not_support_file_type_with_ext">Select a supported %1$s extension file instead.</string>
-    <string name="monitoring_min_speed_descr">This is a low-speed cut-off filter to not record points below a certain speed. This may make recorded tracks look "smoother" when viewed on the map.</string>
-    <string name="monitoring_min_speed_descr_side_effect">Side effect: Your track will be missing all sections where the minimum speed criterion was not met (e.g. where you push your bike up a steep hill). Also, there will be no information about periods at rest, like breaks. This has effects on any analysis or post-processing, like when trying to determine the total length of your trip, time in motion, or your average speed.</string>
-    <string name="monitoring_min_speed_descr_recommendation">Recommendation: Try using the motion detection via the "logging minimum displacement" filter (B) first, it may produce better results, and you will lose less data. If your tracks remain noisy at low speeds, try non-zero values here. Please note that some measurements may not report any speed value at all (some network-based methods), in which case you would not record anything.</string>
```

Comment:
1. Why would (only?) using "minimum logging displacement" be better. "Lose less data" where, in capturing or saving?
2. What is "here"?
3. Some measurements may not report any speed value at all in min speed, or min displacement?